src/app/(profile)/_actions/countProfileView.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  getProfileById,
+  incrementProfileViewCountById,
+} from '@/backend/profile/profile.service'
+import { withSentry } from '@/utils/errHandling'
+
+export const countProfileView = withSentry(
+  async (profileId: string, userId: string) => {
+    const foundProfile = await getProfileById(profileId)
+
+    if (profileId !== userId) {
```

Comment:
**issue:** This condition is wrong, `profileId` is totally different thing than `userId`